network-api/networkapi/wagtailpages/pagemodels/customblocks/image_grid.py

```diff
@@ -2,8 +2,23 @@
 from wagtail.images.blocks import ImageChooserBlock
 
 
+class ImageGridAltText(blocks.StructValue):
+    """Get alt_text or default to image title."""
+
+    def alt(self):
+        alt_text = self.get('alt_text')
+        image = self.get('image')
+        if alt_text:
+            return alt_text
+        return image.title
+
+
 class ImageGrid(blocks.StructBlock):
     image = ImageChooserBlock()
+    alt_text = blocks.CharBlock(
+        required=False,
```

Comment:
tested this on a campaign page by adding an image grid, but publishing still says I have to fill something in:
![image](https://user-images.githubusercontent.com/177243/90054901-5903a080-dc91-11ea-82a0-bd5b17631763.png)